src/components/CheckoutActions/CheckoutActions.js

```diff
@@ -61,13 +65,64 @@ export default class CheckoutActions extends Component {
     cartStore.setStripeToken(stripeToken);
   }
 
+  placeOrder = () => {
+    const { cart, cartStore, placeOrderWithStripeCard } = this.props;
+    const cartId = cartStore.hasAccountCart ? cartStore.accountCartId : cartStore.anonymousCartId;
+    const { email, checkout, shop } = cart;
+    const fulfillmentGroups = checkout.fulfillmentGroups.map((group) => {
+      const address = group.data.shippingAddress;
+      const { selectedFulfillmentOption } = group;
+      const shippingAddress = {
+        address1: address.address1,
+        address2: address.address2,
+        city: address.city,
+        country: address.country,
+        fullName: address.fullName,
+        isCommercial: address.isCommercial,
+        phone: address.phone,
+        postal: address.postal,
+        region: address.region
+      };
```

Comment:
You might not need to rebuild the shipping address like this. I needed to in Meteor, but since yours is from GraphQL, I think it's already correct. You should ideally actually do `data` below without caring whether there is even a shipping address on it because in the future "pickup" groups will not have an address.